app/models/katello/repository.rb

```diff
@@ -767,12 +766,6 @@ def downcase_pulp_id
       end
     end
 
-    def ensure_valid_docker_attributes
```

Comment:
Rather than removing this method I 'd prefer you restore its original version
https://github.com/Katello/katello/commit/2aa2347b7fd0d4b3fee574062863b2855c5a64a0#diff-7bfb4ceeca67af50fe464d5451281e40R494

We want the code say "either set both upstream name and sync url Or dont set either"

Reply:
@parthaa - done, thanks!